### ISSUE TITLE
How to save and re-open turbo intruer attack?

### ISSUE THREAD

**Issue by thedarksource** [PortSwigger/turbo-intruder#94]
Someone, please help me to save the turbo intruder attack which can be re-opened when I want for analysis.
Purpose:
I am using turbo intruder on my server and I want that analysis on my local machine after the scan.

**Comment by albinowax** [comment]
Assuming you're on Burp Pro, you can select requests in the results table and use 'report as issue'. This saves those requests, and the attack script.

**Comment by thedarksource** [comment]
@albinowax Thanks for that suggestion.
I used your suggested option and I got this in my burp's Dashboard
![image](https://user-images.githubusercontent.com/34304093/153405709-eb107811-e07a-4086-8797-67b70f5a1d9e.png)

We can not analyze it if we got tones of requests and responses in the dashboard together as shown in POC.
I want to store the full attack(all requests and responses) like the standard Intruder.

Is there any other way by which we can customize the Turbo intruder to save all the results and afterward we can re-open it from that file?
i.e.: Custom python script.
